.github/workflows/build.yml

```diff
@@ -51,8 +51,8 @@ jobs:
       - name: linting [flake8]
         if: ${{matrix.test-folder == 'tests.tests_geomstats' && matrix.python-version == 3.8 && matrix.geomstats-backend == 'tensorflow'}}
         run: |
-          flake8 --ignore=D,W503,W504 geomstats examples tests
-          flake8 geomstats/geometry geomstats/learning 
+          flake8 --ignore=D,W292,W503,W504, geomstats examples tests
```

Comment:
This is not the most cumbersome convention to respect, and usual IDEs also highlight this, so why do you want to remove it?

Reply:
Will put them back.